.github/workflows/build_and_test.yml

```diff
@@ -473,6 +473,8 @@ jobs:
     runs-on: ubuntu-20.04
     env:
       SPARK_LOCAL_IP: localhost
+    # TODO: Enable this when the data in 'Cache TPC-DS generated data' fits the new schema
```

Comment:
Could you use the ref 556111e35d400f56cb0625dc16e9063d54628320 to checkout  `maropu/spark-tpcds-sf-1`? I updated these schemas and re-generated data in https://github.com/maropu/spark-tpcds-sf-1/commit/556111e35d400f56cb0625dc16e9063d54628320

Reply:
i verifed locally, seemed Okay, thanks @maropu. good job!